src/App.tsx

```diff
@@ -22,8 +23,14 @@ function App() {
         }
       >
         <GlobalStyles />
-        {/* <DashboardPage /> */}
-        <AccessPage />
+        <Switch>
+          <Route exact path="/signup" component={AccessPage} />
```

Comment:
Moved this up here so we really only have two top-level routes, you are either "in" or you still need to "login".